docs/layouts/shortcodes/generated/all_taskmanager_network_section.html

```diff
@@ -27,17 +27,35 @@
             <td>Boolean flag to enable/disable more detailed metrics about inbound/outbound network queue lengths.</td>
         </tr>
         <tr>
-            <td><h5>taskmanager.network.memory.automatic-buffer-adjustment.period</h5></td>
-            <td style="word-wrap: break-word;">500</td>
-            <td>Integer</td>
-            <td>The minimum period of time after which the buffer size will be automatically adjusted to a new value if required. The low value provides a fast reaction to the load fluctuation but can influence the performance.</td>
+            <td><h5>taskmanager.network.memory.buffer-debloat.enabled</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>The switch of the automatic buffered debloating feature. If enabled the amount of in-flight data will be adjusted automatically accordingly to the measured throughput.</td>
         </tr>
         <tr>
             <td><h5>taskmanager.network.memory.automatic-buffer-adjustment.samples</h5></td>
             <td style="word-wrap: break-word;">20</td>
             <td>Integer</td>
             <td>The number of the last buffer size values that will be taken for the correct calculation of the new one.</td>
         </tr>
+        <tr>
+            <td><h5>taskmanager.network.memory.buffer-debloat.enabled</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>The switch of the automatic buffered debloating feature. If enabled the amount of in-flight data will be adjusted automatically accordingly to the measured throughput.</td>
+        </tr>
```

Comment:
Why are there two entires for this? Was it some merge conflict? Does it need to be regenerated once more?

Reply:
thanks. It is artifact after the merge. I will regenerate the doc